aim/sdk/num_utils.py

```diff
@@ -86,23 +98,31 @@ def is_number(value):
     if is_pytorch_tensor(value):
         return True
 
+    if is_tensorflow_tensor(value):
+        return True
+
     return False
 
 
 def convert_to_py_number(value) -> object:
     """
     Converts numpy objects or tensors to python number types
     """
-    if isinstance(value, float):
```

Comment:
please keep the explicit cast to float/int. we had an [issue](https://github.com/aimhubio/aim/pull/1006) with tracking float-compatible types.